<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>JointWidget</class>
 <widget class="QWidget" name="JointWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>250</width>
    <height>21</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <property name="spacing">
    <number>2</number>
   </property>
   <property name="leftMargin">
    <number>2</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>2</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <widget class="QLabel" name="label">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize">
      <size>
       <width>80</width>
       <height>0</height>
      </size>
     </property>
     <property name="text">
      <string>TextLabel</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QDoubleSpinBox" name="angleSpinBox">
     <property name="toolTip">
      <string>The desired angle.</string>
     </property>
     <property name="minimum">
      <double>-3.140000000000000</double>
     </property>
     <property name="maximum">
      <double>3.140000000000000</double>
     </property>
     <property name="singleStep">
      <double>0.010000000000000</double>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QSlider" name="angleSlider">
     <property name="toolTip">
      <string>Analog control of the angle.</string>
     </property>
     <property name="minimum">
      <number>-10000</number>
     </property>
     <property name="maximum">
      <number>10000</number>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QDoubleSpinBox" name="fmaxSpinBox">
     <property name="toolTip">
      <string>Maximum force used to achieve the desired angle.</string>
     </property>
     <property name="maximum">
      <double>9999.000000000000000</double>
     </property>
     <property name="value">
      <double>250.000000000000000</double>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>angleSlider</sender>
   <signal>valueChanged(int)</signal>
   <receiver>JointWidget</receiver>
   <slot>intToAngleSpinBox(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>183</x>
     <y>10</y>
    </hint>
    <hint type="destinationlabel">
     <x>193</x>
     <y>10</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>angleSpinBox</sender>
   <signal>valueChanged(double)</signal>
   <receiver>JointWidget</receiver>
   <slot>doubleToAngleSlider(double)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>107</x>
     <y>9</y>
    </hint>
    <hint type="destinationlabel">
     <x>193</x>
     <y>10</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>intToAngleSpinBox(int)</slot>
  <slot>doubleToAngleSlider(double)</slot>
 </slots>
</ui>
